test/powershell/Modules/Microsoft.PowerShell.Management/FunctionProvider.Tests.ps1

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+Describe "Basic Function Provider Tests" -Tags "CI" {
+    BeforeAll {
+        $existingFunction = "existingFunction"
```

Comment:
I don't see where you are defining the function, `existingFunction`

Reply:
I am defining it in BeforeEach, line 20.